react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+
+export default class Accordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isHtmlOpen: false,
+      isCssOpen: false,
+      isJsOpen: false
+    };
+
+    this.openHTML = this.openHTML.bind(this);
+    this.openCSS = this.openCSS.bind(this);
+    this.openJS = this.openJS.bind(this);
+  }
+
+  openHTML() {
+    this.setState({
+      isHtmlOpen: !this.state.isHtmlOpen,
+      isCssOpen: false,
+      isJsOpen: false
+    });
+  }
+
+  openCSS() {
+    this.setState({
+      isHtmlOpen: false,
+      isCssOpen: !this.state.isCssOpen,
+      isJsOpen: false
+    });
+  }
+
+  openJS() {
+    this.setState({
+      isHtmlOpen: false,
+      isCssOpen: false,
+      isJsOpen: !this.state.isJsOpen
+    });
+  }
```

Comment:
The component shouldn't know about the topics it is showing. Per, the instructions, the topics should be passed as an array of objects via props. You are going to need `.map()` for this solution.